dist/sorts/tablesort.date.min.js

```diff
@@ -1,5 +1,5 @@
 /*!
- * tablesort v5.1.0 (2020-01-22)
+ * tablesort v5.2.0 (2020-06-02)
```

Comment:
@tripu can we make this a patch version instead? `v5.1.1`?

Reply:
Latest published version is `5.2.0`; a patch (`5.2.1`) is what I had in mind, yes.

Reply:
Oh! I see that now from your OP 👍